app/src/main/java/org/mozilla/fenix/components/toolbar/Toolbar.kt

```diff
@@ -124,7 +125,14 @@ class Toolbar(
                 context.getString(R.string.browser_menu_private_tab),
                 R.color.icons
             ) {
-                // TODO Private Tab
+                PreferenceManager.getDefaultSharedPreferences(context)
```

Comment:
I would like to see all of the theme related logic pulled out into its own thing. We can chat about this if you want :)